CONTRIBUTING.md

```diff
@@ -66,16 +66,6 @@ Golden files check that data frames are being generated correctly based on the T
 
 ## Build a release for the Timestream data source plugin
 
-### Pre-requisites
```

Comment:
see https://github.com/grafana/timestream-datasource/issues/240#issuecomment-2316414689 for context on this